docs/howto-update-for-was-fixpack.md

```diff
@@ -0,0 +1,76 @@
+
+   Note: Currently Graham Charters has privilidge to update the image in marketplace, contact him for more information.
+
+1. Do we need to update the solution every time we do the image update?
+   * Yes. That's because image versions of [`twas-nd`](https://github.com/WASdev/azure.websphere-traditional.cluster/blob/master/pom.xml#L51) and [`ihs`](https://github.com/WASdev/azure.websphere-traditional.cluster/blob/master/pom.xml#L57) are explicitely referenced in the tWAS solutoin. Make sure correct image versions are specified in the `pom.xml` of the solution code.
+
+## Updating and publishing the solution code
+
+Note: The steps included in this section are also applied to release new features / bug fixes which have no changes to the images.
+
+1. How to update the version of the solution?
+   * Increase the [version number](https://github.com/WASdev/azure.websphere-traditional.cluster/blob/master/pom.xml#L23) which is specified in the `pom.xml`
+
+1. How to run CI/CD?
+   * Go to [Actions](https://github.com/WASdev/azure.websphere-traditional.cluster/actions) > Click `Package ARM` > Click to expand `Run workflow` > Click `Run workflow` > Refresh the page
+
+1. How to publish the solution in marketplace and who can do it?
+   1. Wait until the CI/CD workflow for `Package ARM` successfully completes > Click to open details of the workflow run > Scroll to the bottom of the page > Click `azure.websphere-traditional.cluster-<version>-arm-assembly` to download the zip file `azure.websphere-traditional.cluster-<version>-arm-assembly.zip`;
+   1. Sign into [Microsoft Partner Center](https://partner.microsoft.com/dashboard/commercial-marketplace/overview) > Click to open the offer for the solution > Clic `Plan overview` > Click to open the plan > Clic `Technical configuration` > Specify the increased version number for `Version` > Click `Remove` to remove the previous package file > Click `browse your file(s)` to upload the downloaded zip package generated by the CI/CD pipeline before > Scroll to the bottom of the page > Click `Save draft` > Click `Review and publish` > Click `Publish`;
+
+   Note: Currently Graham Charters has privilidge to update the image in marketplace, contact him for more information.
+
+1. How to test the solution, what testcases to run?
+   1. Wait until the soluton offer is in `Publisher signoff` (aka "preview") stage;
+   1. Run test cases defined in [twas-solutoin-test-cases.pdf](twas-solutoin-test-cases.pdf). Note: use "preview link" for each test case.
```

Comment:
twas-solutoin-test-cases.pdf -> twas-solution-test-cases.pdf
Fix the file name of the pdf file also.

Reply:
Fixed in 7c7bef3.